erpnext_com/www/docs/user/manual/en/loan-management/loan-interest-accrual.md

```diff
@@ -0,0 +1,27 @@
+<!-- add-breadcrumbs -->
+# Loan Interest Accrual
+> Introduced in Version 13
+
+**Loan Interest will be accrued monthly on first day of every month for demand loans and one day before payment day for term loans.**
+
+<img class="screenshot" alt="Loan Interest Accrual" src="{{docs_base_url}}/assets/img/loan-management/loan-interest-accrual-flow.png">
+
+To access the Loan Interest Accrual list, go to:
+> Home > Loan Management > Disbursement and Repayment > Loan Interest Accrual
+
+<img class="screenshot" alt="Loan Interest Accrual" src="{{docs_base_url}}/assets/img/loan-management/loan-interest-accrual.png">
+
+
+A Loan interest accrual contains the details about:
```

Comment:
better the formatting, for example,
**Pending Principal Amount**: The Principal Amount pending for this Loan

OR

just list them neatly as bullets if the explanation is too trivial